### PR TITLE
Update the AWSCLI pod creation to support disconnected clusters

### PR DESCRIPTION
**Pull request by MeridianExplorer**

### CODE REVIEW

**Review by dahorak — APPROVED** [review]

**Review by petr-balogh — DISMISSED** [review]

File: tests/conftest.py [hunk 1]
```diff
@@ -1782,6 +1783,8 @@ def awscli_pod_fixture(request, scope_name):
     )
     awscli_pod_dict["metadata"]["name"] = awscli_pod_name
 
+    update_container_with_mirrored_image(awscli_pod_dict)
```
Comment (petr-balogh) [inline tests/conftest.py:1786]: BTW should not this be done with condition to run this only when we are running on disconnected cluster?

Comment (dahorak) [reply tests/conftest.py:1786]: The condition is incorporated into the `update_container_with_mirrored_image()` function: https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/utils.py#L2413

Comment (petr-balogh) [reply tests/conftest.py:1786]: After discussion with Daniel I understand that condition is in function laready.

**Review by petr-balogh — APPROVED** [review]

File: tests/conftest.py (hunk 1 above)
Comment (MeridianExplorer) [reply tests/conftest.py:1786]: Thanks for the review and help :) 
Resolved

**Comment by MeridianExplorer** [comment]
[PR3491 verification.txt](https://github.com/red-hat-storage/ocs-ci/files/5667355/PR3491.verification.txt)
Verified locally on a regular cluster. Verified by @dahorak on a disconnected cluster.

**Pull request merged**